CHANGELOG.md

```diff
@@ -58,6 +58,8 @@ and this project adheres to
 
 ### Changed
 
+- The field `password` from user's model can be blank
```

Comment:
No you shouldn't need to do this. When the password is not set, Django sets it to "!" and the user can't log-in with the password.